erroranalysis/erroranalysis/_internal/metrics.py

```diff
@@ -171,6 +171,24 @@ def mean_prediction(y_true, y_pred):
     return np.mean(y_pred)
 
 
+def error_rate(diff, total):
```

Comment:
Can we make its signature similar to other metric functions? 

```
def error_rate(y_pred, y_true):
```

Reply:
diff is already pre-computed, so I think it would waste computation time to do this with y_pred and y_true?

Reply:
From an API point of view the signature of the method is consistent with other metric functions and there is more logic needed in the calling code to use this function. If you really want to use `diff` (which is pre-computed) why don't we change the signature of the method to something like below:-

```
def error_rate(y_pred, y_true, diff=None):
```
That way you can use diff when available and compute error rate from first principle when diff is not available.

Reply:
done, I updated to use this new suggested API instead